content/examples/01-next-app/06-assistants/01-stream-assistant-responses.mdx

```diff
@@ -138,25 +169,31 @@ export async function getAnswer(question: string): Promise<ClientMessage> {
             data.delta.content?.map(part => {
               if (part.type === 'text') {
                 if (part.text) {
-                  text.append(part.text.value);
+                  textStream.append(part.text.value as string);
                 }
               }
             });
           } else if (event === 'thread.run.failed') {
-            console.log(data);
+            console.error(data);
           }
         }
       }
     }
 
-    status.done();
-    text.done();
+    statusUIStream.done();
+    textStream.done();
   })();
 
   return {
     id: nanoid(),
-    status: status.value,
-    text: text.value,
+    status: statusUIStream.value,
+    text: textUIStream.value,
   };
 }
+
+export const AI = createAI({
```

Comment:
q: do we want to add any additional context around where you'd put `<AI>`?

Reply:
makes sense, added additional context to the example!